test/tests/rackhd20/test_rackhd20_api_workflows.py

```diff
@@ -69,5 +70,143 @@ def test_api_20_workflows_put_ipmi(self):
         api_data = fit_common.rackhdapi("/api/2.0/workflows/graphs/" + 'Graph.Obm.Ipmi.CreateSettings.Test')
         self.assertEqual(api_data['status'], 200, 'Incorrect HTTP return code, expected 200, got:' + str(api_data['status']))
 
+    def test_api_20_workflows_waitOn_anyOf(self):
+        data_payload = {
+            "friendlyName": "Multi-REST-call",
+            "injectableName": "Graph.REST.call",
+            "tasks": [
+                {
+                    "label": "REST-call-good",
+                    "taskDefinition": {
+                        "friendlyName": "good REST call",
+                        "injectableName": "Task.Rest.GET",
+                        "implementsTask": "Task.Base.Rest",
+                        "options": {
+                            "url": "http://localhost:8080/api/1.1/nodes",
+                            "method": "GET"
+                        },
+                        "properties": {}
+                    }
+                },
+                {
+                    "label": "REST-call-2",
+                    "taskDefinition": {
+                        "friendlyName": "2nd REST call",
+                        "injectableName": "Task.Rest.GET",
+                        "implementsTask": "Task.Base.Rest",
+                        "options": {
+                            "url": "http://localhost:8080/api/1.1/nodes",
+                            "method": "GET"
+                        },
+                        "properties": {}
+                    }
+                },
+                {
+                    "label": "REST-call-bad",
+                    "taskDefinition": {
+                        "friendlyName": "bad REST call",
+                        "injectableName": "Task.Rest.GET",
+                        "implementsTask": "Task.Base.Rest",
+                        "options": {
+                            "url": "http://localhost:8080/api/never/happen",
+                            "method": "GET"
+                        },
+                        "properties": {}
+                    }
+                },
+                {
+                    "label": "REST-call-final-good",
+                    "taskDefinition": {
+                        "friendlyName": "final REST call",
+                        "injectableName": "Task.Rest.GET",
+                        "implementsTask": "Task.Base.Rest",
+                        "options": {
+                            "url": "http://localhost:8080/api/1.1/nodes",
```

Comment:
/api/1.1 is deprecated.
Why are you hard coding this URL - is it always expected "http://localhost:8080 is valid?  
Sorry, I don't understand the test fully. :-)